### PR TITLE
[SOAR-17751] Cisco Firepower Management Center Snyk Vulnerabilities and SDK Bump

### PR DESCRIPTION
**Pull request by rmurray-r7**
### Description

Cisco Firepower Management Center was failing on a few [snyk vulnerabilities](https://rapid7.atlassian.net/browse/SOAR-17751) which needed the `cryptography` package bumped. SDK and Plugin refresh was also performed.
Describe the proposed changes:

  - SDK Bump to 6.1.2 & Plugin Refresh
  - Bumped `cryptography`

Ticket: https://rapid7.atlassian.net/browse/SOAR-17751

### Testing

Ran Snyk internally and there is no vulnerabilities showing:
image

Tested an action locally via Postman to ensure there is no conflicts with the bumped package (passes):
image

### CODE REVIEW

**Comment by rmurray-r7** [comment]
Unaware why lint is failing. Ran black and all files have been formatted. Think this can be ignored

File: plugins/cisco_firepower_management_center/help.md [hunk 1]
```diff
@@ -706,26 +706,18 @@ Example output:
...
-* 2.1.2 - `Block URL Policy` - Refactor the action to use local API calls | Remove vulnerable dependencies
-* 2.1.1 - Updated to latest SDK version | Fixed issue related to pagination
-* 2.1.0 - `Check if Address in Group`: Extended search for manually added literals | Added new output field `literal_objects`
-* 2.0.1 - Fix issue in Add Address to Group action where Network Groups that had no objects would result in action failure
-* 2.0.0 - Combine Cisco Firepower and Cisco Firepower Management Center plugins
-* 1.2.0 - New actions - Check If Address in Group, Add Address to Group, Remove Address from Group
-* 1.1.0 - New actions - Create Address Object, Delete Address Object
-* 1.0.1 - New spec and help.md format for the Extension Library
-* 1.0.0 - Initial plugin
+* 2.1.3 - Bumped 'cryptography' | SDK Bump to 6.1.2
```
Comment (ekelly-r7) [inline plugins/cisco_firepower_management_center/help.md:714]: Missing some version_history here in the plugin.spec.yaml

Comment (rmurray-r7) [reply plugins/cisco_firepower_management_center/help.md:714]: Well spotted! Don't know how I managed to miss that 🤨

**Review by cmcnally-r7 — COMMENTED** [review]
Fix the indentation in the plugin spec but other than that lgtm

**Review by cmcnally-r7 — APPROVED** [review]

**Review by ekelly-r7 — APPROVED** [review]

**Pull request merged**